src/backend/distributed/sql/udfs/get_cluster_clock/latest.sql

```diff
@@ -0,0 +1,20 @@
+ALTER TABLE pg_catalog.pg_dist_local_group ADD COLUMN logical_clock_value BIGINT NOT NULL DEFAULT 0;
+
+UPDATE pg_catalog.pg_dist_local_group SET logical_clock_value = extract(epoch from now()) * 1000;
+
```

Comment:
we should move these two to citus--11.0-2--11.1-1.sql, given that these are not part of the func def

Reply:
before the line: `#include "udfs/get_cluster_clock/11.1-1.sql"`

Reply:
Thanks, I wasn't sure about this process, it's clear now.